src/snowflake/cli/_plugins/nativeapp/entities/application.py

```diff
@@ -118,6 +118,35 @@ class ApplicationEntity(EntityBase[ApplicationEntityModel]):
     A Native App application object, created from an application package.
     """
 
+    @property
+    def project_root(self):
+        return self._workspace_ctx.project_root
+
+    @property
+    def package_entity_id(self):
```

Comment:
is this guaranteed to be a valid SF identifier? If not, should it be? Same question for all identifier properties here.

Reply:
no, this is the entity ID from snowflake.yml